geomesa-accumulo/geomesa-accumulo-spark/src/main/scala/org/locationtech/geomesa/spark/accumulo/AccumuloSpatialRDDProvider.scala

```diff
@@ -48,38 +45,9 @@ class AccumuloSpatialRDDProvider extends SpatialRDDProvider with LazyLogging {
       if (ds == null || sft == null || qp.isInstanceOf[EmptyPlan]) {
         sc.emptyRDD[SimpleFeature]
       } else {
-        val job = new Job(conf)
-        GeoMesaAccumuloInputFormat.configure(job, paramsAsJava, qp)
-        // We soon want to call this
-        // sc.newAPIHadoopRDD(conf, classOf[GeoMesaAccumuloInputFormat], classOf[Text], classOf[SimpleFeature]).map(U => U._2)
-        // But we need access to the JobConf that this creates internally and doesn't expose, so we repeat (most of) the code here
-
-        // From sc.newAPIHadoopRDD, but can't implement this here
-        // assertNotStopped()
-
-        // From sc.newAPIHadoopRDD
-        // Add necessary security credentials to the JobConf. Required to access secure HDFS.
-        val jconf = job.getConfiguration.asInstanceOf[JobConf]
-        SparkHadoopUtilShim.addCredentials(jconf)
-
-        // Iterate over tokens in credentials and add the Accumulo one to the configuration directly
-        // This is because the credentials seem to disappear between here and the YARN executor
-        // See https://stackoverflow.com/questions/44525351/delegation-tokens-with-accumulo-spark
```

Comment:
Ok, so whatever this approach was doing is unused?

Reply:
Yeah, I asked James S about this in gitter. We removed the code that read this token back out a while ago, but never removed the part that puts it in here. I believe accumulo sets the delegation token and reads it out without us having to explicitly do anything.

Reply:
that stack overflow link has disappeared too, so i'm not sure what was the original intent here :(